styles/mixins/spacing.js

```diff
@@ -8,7 +8,8 @@ module.exports = function (config) {
 
     for (const breakpoint of spacingBreakpoints) {
       styles[breakpoint] = {};
-      styles[breakpoint][`${prop}-${position}`] = config.spacing[breakpoint][size];
+      const sizeKey = size === '0' ? 'none' : size;
```

Comment:
When I leave out this check at line 11, it churns out an undefined node.  The output looks right if I leave it in.